src/accelerate/utils/fsdp_utils.py

```diff
@@ -32,7 +33,31 @@
 logger = get_logger(__name__)
 
 
-def save_fsdp_model(fsdp_plugin, accelerator, model, output_dir, model_index=0):
+def _is_peft_model(model):
+    if is_peft_available():
+        from peft import PeftModel
+    return is_peft_available() and isinstance(extract_model_from_parallel(model), PeftModel)
```

Comment:
Let's move this entire function to `utils.modeling` please, as that makes the most logical sense for it

Reply:
Done.